examples/tx/balances_transfer.eg.ts

```diff
@@ -3,14 +3,16 @@
  * @description Transfer some funds from one user to another.
  */
 
-import { westendDev } from "@capi/westend-dev"
+import { WestendDev, westendDev } from "@capi/westend-dev"
 import { assert } from "asserts"
 import { createDevUsers, is } from "capi"
 import { signature } from "capi/patterns/signature/polkadot"
 
 /// Create two dev users. Alexa will send the funds to Billy.
 const { alexa, billy } = await createDevUsers()
 
+type X = WestendDev
```

Comment:
Leftover from debugging?

Reply:
No it's clearly an integral part of the example